chore(blog) update time-series blog post (#1365)
questdb/questdb.io: blog/2023-01-17-time-series-data.mdx

---
title: What Is Time-Series Data?
author: Nicolas Hourcard
author_title: Co-founder & CEO, QuestDB
author_url: https://github.com/tris0laris
author_image_url: https://avatars.githubusercontent.com/tris0laris

> description: An article on time-series data and time-series databases (TSDB)

Reviewer: I also rewrite the following titles:

Time-series databases design > The design of a time-series database
Time-series databases' most important features and trade-offs > Most important features and trade-offs for a time-series database

Finally, I would delete the "Time series data and characteristics of TSDBs#"
 paragraph entirely because it is redundant.

image: /img/blog/2023-01-17/banner.jpg
tags:
  [
    time-series,

>     startup,

Reviewer: i would remove startup

>     startup,

Reviewer 2: ```suggestion
```

>     entrepreneurship,

Reviewer: remove entrepreneurship

>     entrepreneurship,

Reviewer 2: ```suggestion
```

>     story,

Reviewer: remove story

>     story,

Reviewer 2: ```suggestion
```

  on <a href="https://unsplash.com">Unsplash</a>
</Banner>

This blog post covers the basics of time-series data and its most common use

> cases and applications. We introduce time-series database category, which have

Reviewer: We introduce time-series database category, which have > We introduce the time-series database category, which has

> cases and applications. We introduce time-series database category, which have

Reviewer 2: ```suggestion
cases and applications. We introduce the time-series database category, which has
```

seen an explosion in popularity since a decade ago.

Additionally, we will cover the design and trade-off for a time-series database
and show how SQL can deal with time-series data workloads, using QuestDB SQL

> syntax as an example.

Reviewer: using QuestDB SQL syntax as an example. > using QuestDB's ANSI SQL syntax as an example.

> and show how SQL can deal with time-series data workloads, using QuestDB SQL
> syntax as an example.

Reviewer 2: ```suggestion
and show how SQL can deal with time-series data workloads, using QuestDB's ANSI SQL syntax as an example.
```

<!--truncate-->

## Time-series data explained

> Time series is a succession of data points ordered by time. Time-series data is

Reviewer: Time series is a succession of data points ordered by time. > Time-series is a succession of data points ordered by time

> Time series is a succession of data points ordered by time. Time-series data is

Reviewer 2: ```suggestion
Time-series is a succession of data points ordered by time. Time-series data is
```

This blog post covers the basics of time-series data and why time-series
databases have seen such an explosion in popularity since the category emerged a
decade ago. Additionally, we will cover how QuestDB's SQL provides native
support for processing time-series data.

> 

Reviewer: Change intro paragraph to:

This blog post covers the basics of time-series data and its most common use cases 
and applications. We introduce time-series database category, which 
have seen an explosion in popularity since a decade ago. 
Additionally, we will cover the design and trade-off for a time-series database and
show how SQL can deal with time-series data workloads, using QuestDB SQL syntax as 
an example.
